### DISCUSSION TITLE
view count ticker

### DISCUSSION THREAD

**Discussion by alxndr** [kglw-dot-net/site#7] [Ideas]
a la 1990s

**Comment by alxndr** [comment]
maybe also an "under construction" gif? 😆

**Reply by Hina-Iluvataro** [reply]
100% 
![construction-worker-animation](https://user-images.githubusercontent.com/117538676/200135834-8945c545-03ee-4b98-ba07-c58c519409ee.gif)

**Comment by ClimbedSoSlowly** [comment]
Whether it's visible or not, I'd like to know our pageviews. -Beef
